templates/cassiopeia/index.php

```diff
@@ -28,6 +28,17 @@
 $menu     = $app->getMenu()->getActive();
 $pageclass = $menu->getParams()->get('pageclass_sfx');
 
+// Template path
+$templatePath = Uri::root() . 'templates/'.$this->template;
+
+// Color Theme
+$paramsColorName = $this->params->get('colorName');
```

Comment:
Please set an default parameter

Reply:
Should I add an if statement before as on the fonts part?

Reply:
```$this->params->get('colorName', 'summer');```